src/cryolike/cross_correlation_likelihood.py

```diff
@@ -720,9 +720,11 @@ def compute_cross_correlation_complete(self,
                 in the return
 
         Returns:
-            CrossCorrelationReturn | tuple(CrossCorrelationReturn, integrated_log_likelihood_type): The
-                full cross correlation matrix for each image (if no ILL requested) or a tuple of (same, ILL). ILL is
-                represented as a matrix of scores indexed as [image, template].
+            The full cross correlation matrix for each image (if no ILL requested) or a tuple
+            of (same, ILL). ILL is represented as a matrix of scores indexed as [image, template].
+
+        Return Type:
+            CrossCorrelationReturn | tuple[CrossCorrelationReturn, integrated_log_likelihood_type]
```

Comment:
Note: This is an edit to a different docstring format that makes the automatic documentation generator do the right thing. I did it here to try it out; it'll need to be applied everywhere, but I should do that upstream (in EMPM) rather than in CryoLike. And not as part of this pull request.